src/styles.scss

```diff
@@ -209,7 +209,7 @@ body {
    display: none !important;
 }
 
-@media (max-width: 460px) {
```

Comment:
Do you think this is a good solution?
See screenshots how page looks with the width 375px - first is before the change, second is after.

<img width="439" alt="Screenshot 2024-10-14 at 19 32 46" src="https://github.com/user-attachments/assets/a08236e2-8510-47c8-88f9-83f100afbaa0">
<img width="439" alt="Screenshot 2024-10-14 at 19 34 10" src="https://github.com/user-attachments/assets/ef085189-a864-4f48-8b64-ca0b5cf858ed">